packages/browser/test/index.test.ts

```diff
@@ -151,4 +151,12 @@ describe('browser notifier', () => {
       done()
     })
   })
+
+  it('indicates whether or not the client is started', (done) => {
+    const Bugsnag = getBugsnag()
+    expect(Bugsnag.isStarted()).toBe(false)
+    Bugsnag.start(API_KEY)
+    expect(Bugsnag.isStarted()).toBe(true)
+    done()
+  })
```

Comment:
The `done` function is only needed when writing an asynchronous test that doesn't use promises, so it's not necessary here

```suggestion
  it('indicates whether or not the client is started', () => {
    const Bugsnag = getBugsnag()
    expect(Bugsnag.isStarted()).toBe(false)
    Bugsnag.start(API_KEY)
    expect(Bugsnag.isStarted()).toBe(true)
  })
```